epoch.go

```diff
@@ -0,0 +1,126 @@
+package tiledb
+
+import "time"
+
+const secondsInCommonYear = 31536000
+const secondsInLeapYear = 31622400
+const secondsInDay = 24 * 60 * 60
+const secondsInHour = 60 * 60
+const secondsInMin = 60
+const epochYear = 1970
+
+func isLeapYear(year int) bool {
+	leapFlag := false
+	if year%4 == 0 {
+		if year%100 == 0 {
+			if year%400 == 0 {
+				leapFlag = true
+			} else {
+				leapFlag = false
+			}
+		} else {
+			leapFlag = true
+		}
+	} else {
+		leapFlag = false
+	}
+	return leapFlag
+}
+
+func secondsFromEpochYears(numOfYears int64) int64 {
+	var numOfSeconds int64 = 0
+	for y := int64(epochYear); y < epochYear+numOfYears; y++ {
```

Comment:
I forgot to mention we need to handle negative values too, this is how we get pre-epoch time.

Reply:
Ok